src/FileDeliveryTable.cpp

```diff
@@ -297,7 +301,7 @@ auto LibFlute::FileDeliveryTable::to_string() const -> std::string {
   root->SetAttribute("FEC-OTI-FEC-Encoding-ID", (unsigned)_global_fec_oti.encoding_id);
   root->SetAttribute("FEC-OTI-Maximum-Source-Block-Length", (unsigned)_global_fec_oti.max_source_block_length);
   root->SetAttribute("FEC-OTI-Encoding-Symbol-Length", (unsigned)_global_fec_oti.encoding_symbol_length);
-  root->SetAttribute("xmlns:mbms2007", "urn:3GPP:metadata:2007:MBMS:FLUTE:FDT");
+  root->SetAttribute("xmlns:mbms2007", "urn:3GPP:metadata:2007:MBMS:FLUTE:FDT"); // 3GPP TS 26.346 Clause 7.2.10.2
```

Comment:
It looks incorrect to set this name space on the root element of any XML instance document.

Reply:
That is declaring the "mbms2007" prefix as a shorthand for that namespace. This prefix is used later on in the document for the Cache-Control element and it is quite right to set it here. This is not setting the default namespace for the document.

Reply:
Ah.., Got it. Thanks.